lib/usb-hid-msc-lib.robot

```diff
@@ -0,0 +1,7 @@
+*** Keywords ***
+Upload And Mount DTS Flash ISO
+    [Documentation]    Mounts a bootable ISO as flash USB. Currently
+    ...    only the Qubes OS ISO seems to work for the platform.
+    Upload Image    ${PIKVM_IP}
```

Comment:
@filipleple Please update pikvm submodule. The name of this keyword is currently: `Upload Image To PiKVM` in the main branch as far as I can see....

Reply:
Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)